internal/options/run_options.go

```diff
@@ -3,7 +3,7 @@ package options
 import (
 	"time"
 
-	"github.com/form3tech-oss/f1/pkg/f1/logging"
+	"github.com/form3tech-oss/f1/internal/logging"
```

Comment:
Could you check if the FPS gateway load test uses this? I think they might be overriding the logger.